wikidp/controller.py

```diff
@@ -72,7 +72,7 @@ def search_results_page():
 def preview_selected_page():
     """Show a preview of a selected search result."""
     options = json.loads(request.form['optionList'])
-    print (options)
+    # print (options)
```

Comment:
We should get use to converting these to log statements or remove altogether. Commented code becomes untidy.

Reply:
Agreed! Just removed other instances of this as well in newer commit